packages/babel-config/src/common.ts

```diff
@@ -162,10 +162,7 @@ export const getPathsFromTypeScriptConfig = (
       continue
     }
     const aliasKey = key.replace('/*', '')
-    const aliasValue = path.join(
-      absoluteBase,
-      (value as string)[0].replace('/*', ''),
-    )
+    const aliasValue = path.join(absoluteBase, value[0].replace('/*', ''))
```

Comment:
I think this highlights that the type definition above is a little wrong?

```ts
type CompilerOptionsForPaths = {
  compilerOptions: { baseUrl: string; paths: string }
}
```
I might have thought that `paths` would be a record of strings and not just a string itself?